clients/client-rds/src/runtimeConfig.shared.ts

```diff
@@ -18,6 +21,14 @@ export const getRuntimeConfig = (config: RDSClientConfig) => {
     disableHostPrefix: config?.disableHostPrefix ?? false,
     endpointProvider: config?.endpointProvider ?? defaultEndpointResolver,
     extensions: config?.extensions ?? [],
+    httpAuthSchemeProvider: config?.httpAuthSchemeProvider ?? defaultRDSHttpAuthSchemeProvider,
+    httpAuthSchemes: config?.httpAuthSchemes ?? [
+      {
+        schemeId: "aws.auth#sigv4",
+        identityProvider: (ipc: IdentityProviderConfig) => ipc.getIdentityProvider("aws.auth#sigv4"),
+        signer: new AWSSDKSigV4Signer(),
```

Comment:
a bit late to address this, but this class should be called `AwsSdkSigV4Signer`. We don't capitalize other abbreviations like in `HttpHandler`

Reply:
```js
/**
 * @deprecated renamed.
 */
export const AWSSDKSigV4Signer = AwsSdkSigV4Signer;
```

Reply:
See: https://github.com/aws/aws-sdk-js-v3/pull/5667